go/teams/rpc_exim.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// Export-Import for RPC for Teams
+
+package teams
+
+import (
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func (t *Team) ExportToTeamPlusAllKeys(idTime keybase1.Time) keybase1.TeamPlusAllKeys {
+	var perTeamKeys = make(map[int]keybase1.PerTeamKey)
+	var err error
+	var i = 0
+	for err == nil {
+		perTeamKeys[i], err = t.Chain.GetPerTeamKeyAtGeneration(i)
+		i++
+	}
```

Comment:
Yes it starts at `1`. I'd just add `TeamSigChainState.GetPerTeamKeyMap()` that returns the inner map.